project/frontend/src/components/FormItem.css

```diff
@@ -0,0 +1,30 @@
+.row {
+  display: flex;
+  align-items: center;
+  margin: 16px 0px;
+}
```

Comment:
We should make these class names more specific. Unfortunately importing styles in React isn't as modular as it would seem. Every class we make just gets thrown into the `<head>` section of the html page. So any classnames that are the same will overwrite each other depending on which gets imported last.

What I've been doing is basing class names on the component name, so like `form-item-row` would be better for this.